// this is used for doxygen only.
#error

/// \defgroup messages ROS Messages
/// ROS messages and services defined and used by udp_bridge

namespace udp_bridge
{
/// See: <a href="msg/BridgeInfo.html">udp_bridge/BridgeInfo</a>.
/// \ingroup packet messages
class BridgeInfo{};

/// See: <a href="msg/MessageInternal.html">udp_bridge/MessageInternal</a>.
/// \ingroup packet messages
class MessageInternal{};

/// See: <a href="msg/Remote.html">udp_bridge/Remote</a>.
/// \ingroup messages
class Remote{};

/// See: <a href="msg/RemoteConnection.html">udp_bridge/RemoteConnection</a>.
/// \ingroup messages
class RemoteConnection{};

/// See: <a href="msg/RemoteSubscribeInternal.html">udp_bridge/RemoteSubscribeInternal</a>.
/// \ingroup packet messages
class RemoteSubscribeInternal{};

/// See: <a href="msg/ResendRequest.html">udp_bridge/ResendRequest</a>.
/// \ingroup packet messages
class ResendRequest{};

/// See: <a href="msg/TopicInfo.html">udp_bridge/TopicInfo</a>.
/// \ingroup messages
class TopicInfo{};

/// See: <a href="msg/TopicStatistics.html">udp_bridge/TopicStatistics</a>.
/// \ingroup messages
class TopicStatistics{};



/// See: <a href="srv/AddRemote.html">udp_bridge/AddRemote</a>.
/// \ingroup messages
class AddRemote{};

/// See: <a href="srv/ListRemotes.html">udp_bridge/ListRemotes</a>.
/// \ingroup messages
class ListRemotes{};

/// See: <a href="srv/Subscribe.html">udp_bridge/Subscribe</a>.
/// \ingroup messages
class Subscribe{};

}
